docker-for-mac/edge-release-notes.md

```diff
@@ -11,6 +11,33 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.3.6.0
+2020-09-01
+
+> [Download](https://desktop.docker.com/mac/edge/TODO/Docker.dmg)
+
+### New
+
+- Docker Desktop introduces a new image scanning feature thanks to the partnership between Docker and Snyk.
```

Comment:
Suggest: Docker Desktop introduces the new vulnerability scanning for Docker local images feature, thanks to the partnership between Docker and Snyk.